src/client/src/components/form/formInput.js

```diff
@@ -31,6 +31,31 @@ export const FormInput = props => {
     }
   };
 
+  const isValidDate = dateString => {
+    var regex = /^(\d{4})-(\d{2})-(\d{2})(?:T(\d{2}):(\d{2}))?$/;
+    var matches = dateString.match(regex);
+    if (!matches) {
+      return false;
+    }
+
+    var year = parseInt(matches[1], 10);
+    var month = parseInt(matches[2], 10) - 1;
+    var day = parseInt(matches[3], 10);
+    var hour = matches[4] ? parseInt(matches[4], 10) : 0;
+    var minute = matches[5] ? parseInt(matches[5], 10) : 0;
+
+    var date = new Date(year, month, day, hour, minute);
```

Comment:
Q: Kann man das Ganze nicht vereinfachen, in dem wir gleich prüfen, ob `new Date(dateString)` ein gültiges Datum zurückgibt oder nicht?